### DISCUSSION TITLE
setting arbitary value for background gives module not found error

### DISCUSSION THREAD

**Discussion by shivamchandra75** [tailwindlabs/tailwindcss#8877] [Help] [answered]
Please read the problem here
https://stackoverflow.com/questions/72949611/react-giving-error-on-tailwind-class-bg-url-images-bg-header-mobile-svg?noredirect=1#comment128891967_72949611

**Comment by ichenglin** [comment, marked as answer]
According to your **comment** on **StackOverFlow**, you have your image located at **`/public/images/bg-header-mobile.svg`**.
> [shivam chandra](https://stackoverflow.com/users/15204374/shivam-chandra) asked Jul 12 at 8:59 [(click for original)](https://stackoverflow.com/questions/72949611/react-giving-error-on-tailwind-class-bg-url-images-bg-header-mobile-svg?noredirect=1#comment128891967_72949611)
>
> The file structure is like this post
> [Module not found: Can't resolve '../images/bg-header-desktop.svg' when importing image into React project](https://stackoverflow.com/questions/62020829/module-not-found-cant-resolve-images-bg-header-desktop-svg-when-importing)

I encountered **similar issue** long time ago,
trying to **import** an image from **public** folder into the **source** folder.

My solution is to specify **`/public`** infront of the image source,
so instead of using **`bg-[url('/images/bg-header-mobile.svg')]`**,
replace it with **`bg-[url('/public/images/bg-header-mobile.svg')]`**

I've **reproduced** this solution on my **local machine**,
and I could confirm it **works**, please let me know if it works for you as well.

**Reply by ichenglin** [reply]
An update to this **solution**, I've **cloned** your [repository](https://github.com/shivamchandra75/static-job-listing-master), and applied the solution.

It works 👍 

Here's the result after I executed **`npm run start`** command.

![image](https://user-images.githubusercontent.com/41904540/179471487-386b8f8c-6d26-4e8d-a219-d70d414912f8.png)

And this is what I **changed** for it to work.

![image](https://user-images.githubusercontent.com/41904540/179471577-1a12e481-9619-4889-8849-8f57c15dc898.png)

**Reply by shivamchandra75** [reply]
yes it worked for me as well. Thank you 
So in tailwind we have to specify the public folder in path like this '/public/images/bg-header-mobile.svg' 
unlike in react where it works without specifying the public folder like this '/images/bg-header-mobile.svg'

**Reply by billsbooth** [reply]
Using `/public..` silently fails, using `../public` fails to load module; this is particularly if you're using dynamically generated background image URLs (i.e. on render)
.
